react/README.md

```diff
@@ -4,16 +4,312 @@
 
 ## Table of Contents
 
-  1. [Basic Rules](#basic-rules) #TBD
-  1. [Naming](#naming) #TBD
-  1. [Declaration](#declaration) #TBD
-  1. [Alignment](#alignment) #TBD
-  1. [Quotes](#quotes) #TBD
-  1. [Spacing](#spacing) #TBD
-  1. [Props](#props) #TBD
-  1. [Parentheses](#parentheses) #TBD
-  1. [Tags](#tags) #TBD
-  1. [Methods](#methods) #TBD
-  1. [Ordering](#ordering) #TBD
+  1. [Basic Rules](#basic-rules)
+  1. [Naming](#naming)
+  1. [Declaration](#declaration)
+  1. [Alignment](#alignment)
+  1. [Quotes](#quotes)
+  1. [Spacing](#spacing)
+  1. [Props](#props)
+  1. [Parentheses](#parentheses)
+  1. [Tags](#tags)
+  1. [Methods](#methods)
+  1. [Ordering](#ordering)
+
+## Basic Rules
+
+  - Only include one React component per file.
+  - Always use JSX syntax.
+  - Do not use `React.createElement` unless you're initializing the app from a file that is not JSX.
```

Comment:
Should be changed to always use `React.createElement` and outline the cons of using the class syntax:
* No mixins (although not a really strong argument now that we don't really need them with the exception of `ReactError`)
* Have to define PropTypes and other things after the class declaration unless you use some stage 0-2 proposals
* Others?